hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -842,6 +847,17 @@ public DelegatingSSLSocketFactory.SSLChannelMode getPreferredSSLFactoryOption()
     return getEnum(FS_AZURE_SSL_CHANNEL_MODE_KEY, DEFAULT_FS_AZURE_SSL_CHANNEL_MODE);
   }
 
+  /**
+   * @return Config to select netlib for server communication.
+   */
+  public HttpOperationType getPreferredHttpOperationType() {
+    return getEnum(FS_AZURE_NETWORKING_LIBRARY, DEFAULT_NETWORKING_LIBRARY);
```

Comment:
Since, its more desirable client, Apache shall be the default. However, there are fallback mecahnism in place, which would make the process fallback to JDK, in case the new library is facing some issue. Current mechanism is :
any request on abfsRestOperation which fails more than 3 times (configurable), it will make the whole process fallback to JDK.